fenix/app/src/main/res/values/strings.xml

```diff
@@ -380,8 +380,16 @@
     <string name="preferences_search">Search</string>
     <!-- Preference for settings related to Search engines -->
     <string name="preferences_search_engines">Search engines</string>
+    <!-- Preference for settings related to Search engines suggestions-->
+    <string name="preferences_search_engines_suggestions">Suggestions from search engines</string>
     <!-- Preference for settings related to Search address bar -->
     <string name="preferences_search_address_bar">Address bar</string>
+    <!-- Preference Category for settings related to Search address bar -->
+    <string name="preferences_settings_address_bar" moz:removedIn="121" tools:ignore="UnusedResources">Address bar preferences</string>
```

Comment:
```suggestion
    <string name="preferences_settings_address_bar" moz:removedIn="120" tools:ignore="UnusedResources">Address bar preferences</string>
```
I goofed here and read the calendar wrong

Reply:
realized i also goofed since it's the line above that's no longer in use